### ISSUE TITLE
Remove chempiles from spraycans

### ISSUE THREAD

**Issue by Maroonenahmoron** [HippieStation/HippieStationdeprecated2020#9912]
I've seen this quite a while now and I'm surprised not a whole lot is done about it, but using spraycans on the floor dumps out more than 40u of ethanol and welding fuel. This in turn can **transfer** to people walking over the floors and racking up a lot of toxin damage, particularly if there's long strings on the ground you walk over at once.

**Comment by DaydreamPuppeh** [comment]
Yeah. This has been an issue for a while. Causes service borgs to unintentionally cause human harm.

**Comment by k-k-karma** [comment]
spray cans should only contain 1 unit of ethanol and welding fuel
